packages/calling-component-bindings/src/errorBarSelector.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getLatestErrors } from './baseSelectors';
+import { ErrorType } from '@internal/react-components';
+import { createSelector } from 'reselect';
+
+/**
+ * Select the first fiew active errors from the state for the `ErrorBar` component.
+ *
+ * In case there are many errors, only a few top errors are returned to avoid
```

Comment:
Can we add absolute numbers here instead of many/few?

Reply:
Done